source/main_cr.f90

```diff
@@ -1321,6 +1322,10 @@ end subroutine envarsv
 
   ! Initialise Modules
   call dump_initialise
+  if(iclo6 > 0 .and. ithick == 0 .and. do_coll) then
```

Comment:
not sure this is the best check in this position...
I guess we should have a general check after `daten` that exits the program if this condition is not satisfied

Reply:
Added the check to the exit call as well, as we discussed.